backend/spherre/app/models/account.py

```
@@ -0,0 +1,70 @@
+    address = db.Column(db.String, unique=True, nullable=False)
+    name = db.Column(db.String(100), nullable=True)
+    description = db.Column(db.String, nullable=True)
+    is_private = db.Column(db.Boolean, default=True)
+    threshold = db.Column(db.Integer)
+    members = db.relationship("Member", secondary=account_members, backref="accounts")
+
+    def __repr__(self):
+        return f"<Account {self.name} >"
+
+
+class Member(ModelMixin, db.Model):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)